README.md

```diff
@@ -34,14 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+ Adaptive sites are quick to create like a fixed layout, use breakpoints to accommodate different devices
+	Also the design is often divided among desktop, tablet, or phone. Media queries are used just like a responsive but the layout widths are still hard coded at each media query.  Responsive sites combines the features of fixed, adaptive and fluid such as percentage widths, flexible box layouts, some pixels. The design is often divided among desktop, tablet, or phone, with the holy advent of LESS/SCSS/SASS we can nest media queries right in with their target, additionally, responsive units are used throughout the site such as rem, em, percentage, viewport height, viewport width.
+
+
 2. Describe what it means to be mobile first vs desktop first.
 
+mobile first: make it fit a phone/tablet/wearable device and work up from there. desktop first: start with a nifty full screen desktop layout that big-boss approved and then make it fit the little people.f
+
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+based on a 16px default browser font size, setting the HTML to 62.5% creates a base of 10px so all percentage in the children then work off of that measurement.(after an exhaustive search I couldn't find an actual conversion ration that could be used with any default browser font size, i.e. what if we started with 20, 22 or even just 18px?)
```

Comment:
all browsers are based off of 16px.  If you need a different ratio, they can also be calculated.  
`10/20px = percentage you want`